### ISSUE TITLE
Backbone plugin?

### ISSUE THREAD

**Issue by manuelma** [spine-tools/Spine-Toolbox#1256]
I think it would be nice to try and create the Backbone plugin so users can install it in their toolbox. 

In principle the plugin would have the main Item specifications needed in a backbone project. I remember seen some importer specs to convert Backbone to SpineOpt, that could be good to have in the plugin as well?

**Comment by jkiviluo** [comment]
Yes, this is very desirable to allow user migration. However, things are still bit unstable. So, I would first try to iron out something that works reasonably well. And then we will really need version control there, since data structures keep changing (either in Backbone or in SpineOpt). That's of course true between any two models.

**Comment by jkiviluo** [comment]
ines-spec, ines-backbone and ines-spineopt repos deal with this. Backbone plugin could still be desirable, but that's Backbone issue.
